doc/source/upgrading_guide.rst

```diff
@@ -94,8 +94,19 @@ If you only used the |MyRocks| storage engine in |Percona Server| |version.prev|
 
    $ apt-get install percona-server-rocksdb
 
+.. deprecated:: 8.0.16-7
 
-The installation script will *NOT* run automatically :command:`mysql_upgrade` as
+The :command:`mysql_upgrade` was deprecated and removed as of |Percona Server|
```

Comment:
I made a mistake - It was NOT REMOVED. Just deprecated. I even put initially the text which the command outputs about deprecation when you run it in the Jira ticket - somehow I have missed this. So please remove the `and removed` part from this sentence in all places.
Thanks!

Reply:
No problem.